### ISSUE TITLE
Clean up repository for Make It So on Apple

### ISSUE THREAD

**Issue by peterfriese** [peterfriese/MakeItSo#115]
* Use `main` as the default branch
* Do feature development on `feature` branches and merge into `main` once ready
* Once implementation work is finished, create separate branches for derivative work (videos, blog posts, interactive DocC tutorials)

- [ ] Create a branch for the current version
- [ ] Create README and point to the previous version(s)

**Comment by mikeymike9000** [comment]
Hi Peter, thanks very much for the example app, it was super helpful to see potential 'best practices'.. I've been wondering if you might be planning a Swift 6 update showing the latest approaches you'd take for firebase auth, states, dependency injection, etc. In other words replace Combine with the new concurrency model to have a hybrid of @Observable for state management and Actors for thread safety, etc. I ask because I've learned so much from you and want this Friese train to keep rollin!

**Comment by peterfriese** [comment]
> Hi Peter, thanks very much for the example app, it was super helpful to see potential 'best practices'.. I've been wondering if you might be planning a Swift 6 update showing the latest approaches you'd take for firebase auth, states, dependency injection, etc. In other words replace Combine with the new concurrency model to have a hybrid of @observable for state management and Actors for thread safety, etc. I ask because I've learned so much from you and want this Friese train to keep rollin!

Hi @mikeymike9000 - yes, we're working on it. Feel free to use the [GitHub discussions for this project](https://github.com/peterfriese/MakeItSo/discussions/categories/q-a) to ask questions.

**Comment by mikeymike9000** [comment]
great thanks Peter and I'll use GitHub discussions going forward!
